apidoc/released/icu4j/com/ibm/icu/lang/UCharacter.html

```diff
@@ -589,14 +589,14 @@ <h3>Method Summary</h3>
 <tr id="i9" class="rowColor">
 <td class="colFirst"><code>static int</code></td>
 <td class="colLast"><code><span class="memberNameLink"><a href="../../../../com/ibm/icu/lang/UCharacter.html#digit-int-">digit</a></span>(int&nbsp;ch)</code>
-<div class="block">Returnss the numeric value of a decimal digit code point.</div>
+<div class="block">Returns the numeric value of a decimal digit code point.</div>
```

Comment:
Thanks, but this is generated javadoc. We will need to fix this in https://github.com/unicode-org/icu/blob/master/icu4j/main/classes/core/src/com/ibm/icu/lang/UCharacter.java